packages/web/src/assets/styles/shadows.css

```diff
@@ -0,0 +1,16 @@
+:root {
+  /*
+   * Default/Near
+   * - onPress state on card elements
+   */
+  --box-shadow-near: 0px 0px 1px -2px var(--tile-shadow-1),
+    0px 1px 0px -2px var(--tile-shadow-2), 0px 2px 5px -2px var(--tile-shadow-3);
```

Comment:
kinda think it's time to have a preset for these shadows. wdyt?

Reply:
yes pls